### PR TITLE
Rename method arguments to use camelCase casing (libraries/src directory)

### PR DESCRIPTION
**Pull request by SharkyKZ**
Partial Pull Request for Issue https://github.com/joomla/joomla-cms/issues/30988.

### Summary of Changes

Renames some method arguments to use camelCase casing.

### Testing Instructions

Thorough code review / Joomla works like before.

### Documentation Changes Required

No.

### CODE REVIEW

File: libraries/src/Filesystem/Stream.php [hunk 1]
```diff
@@ -1082,15 +1082,15 @@ public function appendFilter($filtername, $read_write = STREAM_FILTER_READ, $par
...
+	 * @param   integer  $readWrite   Optional. Defaults to STREAM_FILTER_READ.
 	 * @param   array    $params      An array of params for the stream_filter_prepend call.
 	 *
 	 * @return  mixed
 	 *
 	 * @link    https://www.php.net/manual/en/function.stream-filter-prepend.php
 	 * @since   1.7.0
 	 */
-	public function prependFilter($filtername, $read_write = STREAM_FILTER_READ, $params = array())
+	public function prependFilter($filtername, $readWrite = STREAM_FILTER_READ, $params = array())
```
Comment (HLeithner) [inline libraries/src/Filesystem/Stream.php]: $filterName ?

**Review by HLeithner — APPROVED** [review]
Code Review

File: libraries/src/Filesystem/Stream.php (hunk 1 above)
Comment (SharkyKZ) [reply libraries/src/Filesystem/Stream.php]: Renamed.

**Review by zero-24 — APPROVED** [review]

File: libraries/src/Filesystem/Folder.php [hunk 2]
```diff
@@ -595,19 +595,19 @@ public static function folders($path, $filter = '.', $recurse = false, $full = f
...
-	 * @param   boolean  $full                  True to return the full path to the file.
-	 * @param   array    $exclude               Array with names of files which should not be shown in the result.
-	 * @param   string   $excludefilter_string  Regexp of files to exclude
-	 * @param   boolean  $findfiles             True to read the files, false to read the folders
+	 * @param   string   $path                 The path of the folder to read.
+	 * @param   string   $filter               A filter for file names.
+	 * @param   mixed    $recurse              True to recursively search into sub-folders, or an integer to specify the maximum depth.
+	 * @param   boolean  $full                 True to return the full path to the file.
+	 * @param   array    $exclude              Array with names of files which should not be shown in the result.
+	 * @param   string   $excludefilterString  Regexp of files to exclude
```
Comment (Quy) [inline libraries/src/Filesystem/Folder.php]: `$excludeFilterString`?
`$findFiles`?

Comment (SharkyKZ) [reply libraries/src/Filesystem/Folder.php]: Renamed.

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on e1929e8868827db99740ac503ac1cbf3cceb4979<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31148">issues.joomla.org/tracker/joomla-cms/31148</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on e1929e8868827db99740ac503ac1cbf3cceb4979<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31148">issues.joomla.org/tracker/joomla-cms/31148</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31148">issues.joomla.org/tracker/joomla-cms/31148</a>.</sub>

**Pull request merged**